### PR TITLE
docs: add Comp Labs website to showcase

### PR DESCRIPTION
**Pull request by RudraSen2**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [x] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [x] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation

The Comp Labs Website is made using pure Docusaurus v2. It could be cool to add it to the showcase page.

## Test Plan

Tested docusaurus site by running `yarn run start:v2` and checked http://localhost:3000/showcase is rendered properly.

### Test links

Deploy preview: https://deploy-preview-_____--docusaurus-2.netlify.app/

## Related issues/PRs

None

### CODE REVIEW

**Comment by RudraSen2** [comment]
Deploy Preview: https://deploy-preview-7652--docusaurus-2.netlify.app/

I've actually signed CLA after running checks.

**Comment by Josh-Cena** [comment]
Just wait for a few hours. Not a big deal.

**Comment by RudraSen2** [comment]
I can't make the correct preview image for my site showcase. Can you make it for me according to the rulesets?

Homepage: https://complabs.in

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -444,6 +444,14 @@ const Users: User[] = [
     source: 'https://github.com/CodeYourFuture/syllabus',
     tags: ['opensource', 'product'],
   },
+  {
+    title: 'Comp Labs',
+    description: 'Tech-based Blog, Enterprise-Grade Solutions and more...',
+    preview: require('./showcase/comp-labs.png'),
+    website: 'https://complabs.in/',
+    source: 'https://github.com/Comp-Labs/comp-labs-website',
+    tags: ['opensource', 'design', 'large', 'personal'],
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:453]: 1. This site doesn't seem to have 200+ pages?
2. Is it personal?

Comment (RudraSen2) [reply website/src/data/users.tsx:453]: Yes... I seem to add personal and large tags by mistake and in a hurry. So these tags has to be removed

**Review by Josh-Cena — APPROVED** [review]
Thanks

**Comment by RudraSen2** [comment]
Please add the `design` tag to my site.

**Comment by Josh-Cena** [comment]
Tags are reviewed by the team; the fact that we removed the tag is a signal that we don't think it qualifies. We think it still doesn't—you are basically reusing Infima components and not really using them in the most appeasing way. Plus, the navbar is artifically shrunk and won't pass accessibility tests. Please try to put more effort into making it consistent—existing sites with the "design" tag are good sources of inspiration.

**Pull request merged**